frontend/src/utils/chainsInfo.ts

```diff
@@ -334,6 +334,77 @@ export const networks: Network[] = [
       }
     }
   },
+  {
+    enableModules: {
+      authz: true,
+      feegrant: true,
+      group: false
+    },
+    aminoConfig: {
+      authz: false,
+      feegrant: false,
+      group: false
+    },
+    showAirdrop: true,
+    logos: {
+      toolbar: "https://raw.githubusercontent.com/vitwit/chain-registry/aleem/staking-assets/passage3d/images/passage3d-logo.png",
+      menu: "https://raw.githubusercontent.com/vitwit/chain-registry/aleem/staking-assets/passage3d/images/passage.png"
+    },
+    keplrExperimental: true,
+    leapExperimental: true,
+    isTestnet: false,
+    explorerTxHashEndpoint: "https://passage.aneka.io/txs/",
+    config: {
+      chainId: "passage-testnet-1",
+      chainName: "passage-testnet",
```

Comment:
Can we make this `Passage-Testnet` or let it be `passage-testnet`

Reply:
Rename it to `Passage-testnet`